rosserial_python/src/rosserial_python/SerialClient.py

```diff
@@ -365,33 +371,45 @@ def __init__(self, port=None, baud=57600, timeout=5.0):
 
         self.buffer_out = -1
         self.buffer_in = -1
+        self.resetCallbacks()
+
+        rospy.sleep(2.0) # TODO
```

Comment:
What happens if you get rid of this delay?

Reply:
Unclear .... will test what removing it does when getting other numbers.

Reply:
unclear -- it was already there (previous line 384;) can test affect of removing it.